src/server/game/Entities/Conversation/Conversation.cpp

```diff
@@ -134,6 +136,28 @@ bool Conversation::Create(ObjectGuid::LowType lowGuid, uint32 conversationEntry,
             ++actorsIndex;
     }
 
+    // reset actorsIndex
+    actorsIndex = 0;
+
+    for (ObjectGuid::LowType actorGuids : conversationTemplate->ActorGuids)
+    {
+        if (actorGuids)
+        {
+            auto bounds = map->GetCreatureBySpawnIdStore().equal_range(actorGuids);
+
```

Comment:
What is the best way to get the creature, checking it's alive, compatible with the dynamic_spawn system ?